reproject/mosaicking/background.py

```diff
@@ -0,0 +1,133 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+from __future__ import absolute_import, division, print_function
+
+from math import exp
+
+import numpy as np
+
+__all__ = ['solve_corrections_sgd', 'determine_offset_matrix']
+
+
+def determine_offset_matrix(arrays):
+    """
+    Given a list of ReprojectedArraySubset, determine the offset
```

Comment:
What are `ReprojectedArraySubset`? I can't find their definition.

Reply:
See the ``subset_array.py`` file - this is not meant to be part of the public API.